src/scenes/home/lesson-plan-list.component.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
```

Comment:
This is a new scene right? I think these files should go into a new scene folder (not inside home)

Reply:
@albertjvm fixed!